### ISSUE TITLE
Teleport to Waypoint button gone

### ISSUE THREAD

**Issue by akumasakura** [TeamJM/journeymap#80]
#### JourneyMap Version:

1.12.2-5.5.3

#### Forge Version:

1.12.2-14.23.5.2772

#### Describe the problem in repeatable steps:

1. When I open the waypoint screen the teleport button is gone
2. I tried restarting minecraft, restarting my pc and deopping and then opping myself on the server again
3. The issue is now happening with more people at the server
4. I looked at github, but there it said that the solution was in 5.6.01b, but on the discord they tell me 5.6.x was pulled

#### Relevant errors from `.minecraft/journeymap/journeymap.log`:

none

**Comment by lilithwhite** [comment]
JourneyMap Version:
1.12.2-5.5.3

Forge Version:
1.12.2-14.23.5.2781

Same issue here. It disappeared a while ago after I updated my game and all the mods and I haven't seen that button since (Can't remember the last version it worked on). I figured it was removed until someone on the server commented that they can see it. It does work for a few people on the server though but most OP players can't see the button.
